tests/simulator/per_test.h

```diff
@@ -111,8 +111,12 @@ struct Simulator2dTest : public ::testing::Test
 };
 
 
-using Simulators2d = testing::Types<SimulatorTestParam<2, 1, 8>, SimulatorTestParam<2, 2, 8>,
-                                    SimulatorTestParam<2, 3, 8>>;
+using Simulators2d = testing::Types<
+    SimulatorTestParam<2, 1, 4>, SimulatorTestParam<2, 1, 5>, SimulatorTestParam<2, 1, 8>,
+    SimulatorTestParam<2, 1, 9>, SimulatorTestParam<2, 2, 4>, SimulatorTestParam<2, 2, 5>,
+    SimulatorTestParam<2, 2, 8>, SimulatorTestParam<2, 2, 9>, SimulatorTestParam<2, 2, 16>,
```

Comment:
you might want this

https://github.com/PHAREHUB/PHARE/pull/314/files#diff-175a8ef989f1f57eb9b35db105619ea5R216